likelihood/2pt/2pt_like.py

```diff
@@ -346,7 +346,7 @@ def do_likelihood(self, block):
                 log_det = block[names.data_vector, self.like_name + "_LOG_DET"]
 
             like = -0.5 * log_det - 0.5 * N * np.log(1 + chi2 / (N - 1.))
-
+            
```

Comment:
Could you undo this change, since it doesn't do anything?

Reply:
I deleted this unnecessary indent.

Reply:
I deleted some commented out lines